guides/common/modules/proc_preparing-for-disaster-recovery-by-virtualizing-your-projectserver.adoc

```diff
@@ -0,0 +1,27 @@
+[id="preparing-for-disaster-recovery-by-virtualizing-your-{project-context}-server"]
+= Preparing for disaster recovery by virtualizing your {ProjectServer}
+
+Implement a reliable process for regularly taking VM snapshots of your virtualized {ProjectServer} and for backing up your snapshots for long-term storage.
+
+.Procedure
+. Define a schedule for taking periodic snapshots of your virtualized {ProjectServer}.
+Consider your tolerance for potential data loss:
+Taking snapshots frequently will result in smaller amounts of data loss in case of a disaster.
+However, creating a snapshot takes time, and the snapshots also require storage space.
```

Comment:
The also often affect the performance of the snapshotted guest while being created.

Reply:
Turns out it's good to push, too. So this didn't make it in after all but because it was a nitpick (your words, not mine), I won't worry about it.